### DISCUSSION TITLE
Turning Spotify volume all they way down only gets down to level 17

### DISCUSSION THREAD

**Discussion by rolfost** [rern/rAudio#858] [Bug Reports]
v20220317 with Merus Amp. Tested on a fresh install on both Pi Zero 1 and 2. Tested both on the release image and the UPDATE branch (the 220317 image is missing for Pi Zero 1, so this configuration was tested after updating from the web interface). All tests right after a reboot.

When volume is turned all the way down from Spotify (which would normally mean silence), the sound level out of the speakers is a bit too loud. The web interface shows volume 17 (out of 100).

Mixer = A.Mstr Vol
Volume Control = Mixer device

Equalizer off (if turned on, no sound is coming out at all).

Anything obvious I am overlooking? Or a true bug?

**Comment by rern** [comment]
Try: https://github.com/rern/rAudio-1/discussions/842#discussioncomment-2385261

**Comment by rolfost** [comment]
@rern  Alsamixer can adjust the volume as expected all the way up and down. The web interface can also adjust the volume as expected.

The problem is when volume is adjusted from Spotify. Turning volume to say 5 in alsamixer works fine. If you then turn the volume down from Spotify all the way down you are now at 17.

**Reply by rern** [reply]
Setting volume level with phone buttons is done by Spotify app to `spotifyd`. rAudio just reads the set value and displays accordingly. The on-board audio can go down to 8, not 0, as the lowest.
On the other hand, volume knob/bar on the web interface directly sets volume level with `amixer` command which can go down to 0.

**Reply by rolfost** [reply]
Thanks. So what would need to change for volume to be adjusted to below level 17 from the Spotify app? Or where would the best place to change be?

**Reply by rolfost** [reply]
Repeated the test with the same clean install on the Pi Zero V1 with an Allo Boss HAT. In this setup you are absolutely right. When volume is turned all the day down from Spotify this shows up as level 8 in the web app.

There is a clear difference in the exact same test with the Merus Amp HAT. 

Also, with the Merus Amp level 8 is too loud for background music with the B&W speakers I am using. Level 4-5 is more what I need for that.
